src/components/pages/creation.tsx

```diff
@@ -1,5 +1,99 @@
 function Creation(): JSX.Element {
-  return <div>Creation</div>;
+  return (
+    <div style={{ textAlign: 'left', paddingLeft: '2.5vw' }}>
+      Creation and Deletion
+      <br />
+      <br />
+      At the heart of any project is the ability to create and delete things. So
+      let’s start with making commands.
+      <br />
+      <br />
+      touch
+      <br />
+      The touch command makes a new empty file.
+      <br />
+      Run `touch emptyFile` and see how the file system diagram changes.
+      <br />
+      If you then run `cat emptyFile`, notice how nothing is printed because the
+      file has no contents!
+      <br />
+      <br />
+      mkdir
```

Comment:
Use < h2> for subheadings such as this

Reply:
Also do for any other subheadings

Reply:
I have added them but did not commit them for the same reason as below